lib/signs_ui/signs/expiration.ex

```diff
@@ -0,0 +1,76 @@
+defmodule SignsUI.Signs.Expiration do
+  @moduledoc """
+  Handle cleaning up settings in the configuration that have since expired
+  """
+
+  @type state :: %{
+          time_fetcher: (() -> DateTime.t()),
+          loop_ms: integer(),
+          sign_state: pid()
```

Comment:
I find `sign_state` a confusing name here, since it implies to me the `%{id => sign}` map. In addition, it's not really a `pid` but a `GenServer.server()`. What about `sign_state_server: GenServer.server()` ?

Reply:
Done.